fsspec/implementations/tests/test_local.py

```diff
@@ -527,6 +530,11 @@ def test_linked_files(tmpdir):
     with of as f:
         assert f.read() == data
 
+    # After unlinking source we should still get the info about link
+    os.unlink(fn0)
+    assert fs.info(fn1)
+    assert fs.info(fn2)
```

Comment:
This is probably where you wanted to check that the size of the broken link, that should clearly be not equal to the size of data.